packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md

```diff
@@ -0,0 +1,166 @@
+# ACTION DROPDOWN COMPONENT
+
+This component display a dropdown with items in it. It used react-bootstrap component DropdownButton, MenuItem.
+
+## RENDERING
+
+#### CORE
+The main component used DopdownButton from react-bootstrap
+```javascript
+	<DropdownButton
+			title={title}
+			bsStyle={style}
+			role="button"
+			onSelect={onItemSelect}
+			className={classNames(theme['tc-dropdown-button'], 'tc-dropdown-button')}
+			{...rest}
+		>
+			{chooseMenuItemRendering(getComponent, items, components)}
+		</DropdownButton>
+```
+You can define a title, bsStyle etc...
+
+A onSelect callback can be used 
+```javascript
+	function onItemSelect(object, event) {
+		if (onSelect) {
+			onSelect(event, object);
+		}
+	}
+```
+
+#### OVERLAY
+You can add an overlay to the main dropdown button. It will used TooltipTrigger from react-bootstrap.
+```javascript
+	if (hideLabel || tooltipLabel) {
```

Comment:
not sure component code is usefull, would better love an usage of what hideLabel does by default or what tooltipLabel does. Maybe usage example and link to a story